main/sketch.cpp

```diff
@@ -381,23 +383,23 @@ void setupSabertooth() {
     // On a Sabertooth 2x25, the value is (Desired Volts - 6) X 5.
     // So, in this sample, we'll make the low battery cutoff 12V: (12 - 6) X 5 = 30.
     // sabertoothDome.setMinVoltage(30);
-    // sabertoothDome.setMaxSpeed(100);
-    // sabertoothDome.setUseThrottle(false);
-    // sabertoothDome.setScaling(false);
-    // sabertoothDome.setChannelMixing(false);
-    // sabertoothDome.setDomePosition(&domeSensor);
-
-    // sabertoothTankController.setMinVoltage(30);
-    sabertoothDiff.SetSpeedLimit(0.65);
-    sabertoothDiff.SetSafetyEnabled(true);
+
+    // Setup the Drive motors
+    sabertoothDiff.SetSpeedLimit(C110P_DRIVE_MAXIMUM_SPEED);
+    sabertoothDiff.SetSafetyEnabled(C110P_MOTOR_SAFETY);
+    sabertoothDiff.SetExpiration(C110P_MOTOR_SAFETY_TIMEOUT_MS);
+    sabertoothDiff.SetRampingValue(C110P_DRIVE_RAMPING_PERIOD);
+    sabertoothDiff.SetDeadband(C110P_DRIVE_DEADBAND);
+
     sabertoothDiffDrive.GetMotor(1).SetInverted(true);
     sabertoothDiffDrive.GetMotor(2).SetInverted(false);
```

Comment:
Do you think these parameters should be moved to the SettingsSystem.h or SettingsUsser.h file?

This controls the x/y readings from the joystick before sending it to the motors